tests/integration/cmd_dev_api_server_test.go

```diff
@@ -0,0 +1,73 @@
+package integration
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-developer/odo/tests/helper"
+	"net/http"
+	"path/filepath"
+)
+
+var _ = Describe("odo dev command with api server tests", func() {
+	var cmpName string
+	var commonVar helper.CommonVar
+
+	// This is run before every Spec (It)
+	var _ = BeforeEach(func() {
+		commonVar = helper.CommonBeforeEach()
+		cmpName = helper.RandString(6)
+		helper.Chdir(commonVar.Context)
+		Expect(helper.VerifyFileExists(".odo/env/env.yaml")).To(BeFalse())
+	})
+
+	// This is run after every Spec (It)
+	var _ = AfterEach(func() {
+		helper.CommonAfterEach(commonVar)
+	})
+	for _, podman := range []bool{false, true} {
+		podman := podman
+		for _, customPort := range []bool{false, true} {
+			customPort := customPort
+			When("the component is bootstrapped", func() {
```

Comment:
You need to add `helper.LabelPodmanIf(podman, ` or the test with podman will run on clusters